README.md

```diff
@@ -1,15 +1,20 @@
 # WaveRL
-A package for training RL agents to perform active damping on a model of a oscillating bridge.
+A package for training RL agents to perform active damping on a model of a vibrating bridge.
 
 ## Contents
-* complex_envs/ :  This folder contains the code for the environments as well as scripts for training and rolling out agents
+* src/ :  This folder contains the code for the environments as well as scripts for training and rolling out agents
 	* train.py :  This script trains an agent (see example below)
 	* rollout.py : This script rolls out a trained agent (see example below)
 	* visualize.py : This script produces visualizations of a rolled out agent (see example below)
-	* finite_diff_wave.py : This is a class definition for a simulator of one dimensional wave equation with finite difference methods.
-	* active_damping_env.py : This is a class definition for an OpenAI gym environment simulating an oscillating bridge
+	* environments/ : This folder contains code for the environments.
+		* finite_diff_wave.py : This is a class definition for a simulator of one dimensional wave equation with finite difference methods.
+		* active_damping_env.py : This is a class definition for an OpenAI gym environment simulating an oscillating bridge
 * configs/ 
 	* config.yml : This file holds the default parameters for the scripts and environments
+* tests/
+	* config_test.py :  A unnittest test fixture that can be used to make sure `configs/config.yml` has all the appropriate keys and valid parameter settings
+* trained_agents/ : A folder for storing trained agents
+* rollouts/ : A folder for storing rollouts of trained agents
```

Comment:
It would be good to add sample .gif, .png, and .npz files here so users can see what the output is supposed to be, and for users to be able to take a quick look at the result without having to install and run everything themselves